README.md

```diff
@@ -47,8 +47,7 @@ Internally, the steps involved are:
 
 Following is an example of non-native transfers between Google cloud storage and Sqlite:
 
-https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_universal_transfer_operator.py#L37-L41
-
+https://github.com/astronomer/apache-airflow-provider-transfers/blob/a80dc84b7f33bb86ae244f79411b240f4f4c7e22/example_dags/example_universal_transfer_operator.py#L45-L54
```

Comment:
This example is for S3 to GCS but the above line says GCS and sqlite.

Reply:
Changed: https://github.com/astronomer/apache-airflow-provider-transfers/pull/50/commits/0def63336d40d45f28fbe4b11941805910906b69